pombola/core/management/commands/core_sync_everypolitician_uuid.py

```diff
@@ -0,0 +1,54 @@
+from everypolitician import EveryPolitician
+
+from django.core.management.base import BaseCommand
+
+from pombola.core.models import Person
+
+
+class Command(BaseCommand):
+    help = 'Sync EveryPolitician UUID to Person.everypolitician_uuid field'
+
+    def add_arguments(self, parser):
+        parser.add_argument('everypolitician_countries_json_git_ref',
+                            help="A git ref from the everypolitician-data repo")
+
+    def handle(self, **options):
+        verbose_level = options['verbosity']
+
+        # FIXME: This SHA is currently pointing at a branch which has the
+        # `peoples_assembly` identifier, but this isn't on master because
+        # South Africa is broken in EveryPolitician at the moment.
+        #
+        # TODO: Make this SHA configurable.
+        url_template = ('https://cdn.rawgit.com/everypolitician/everypolitician-data/'
+                        '{git_ref}/countries.json')
+
+        url = url_template.format(git_ref=options['everypolitician_countries_json_git_ref'])
+
+        ep = EveryPolitician(countries_json_url=url)
+        south_africa_assembly = ep.country('South-Africa').legislature('Assembly').popolo()
+
+        id_lookup = {}
+        for popolo_person in south_africa_assembly.persons:
+            pa_ids = [i for i in popolo_person.identifiers if i['scheme'] == 'peoples_assembly']
+            if pa_ids == []:
+                continue
+            id_lookup[pa_ids[0]['identifier']] = popolo_person.id
+
+        error_msg = u"No EveryPolitician UUID found for {} {} https://www.pa.org.za/person/{}/\n"
+        for person in Person.objects.filter(hidden=False):
+            uuid = id_lookup.get(str(person.id))
+            if uuid is None:
+                if verbose_level > 1:
```

Comment:
I'd probably collapse these two conditions into a single condition - perhaps makes it slightly easier to see that there's only one branch here?

Reply:
The `continue` on the line below is actually in the top level `if` statement, then the logging is part of the inner `if` statement. I've changed things around in https://github.com/mysociety/pombola/pull/2331/commits/663e8dc2af62e5c38f490debc34ed0c423943db7 so that it's all at a single level of `if` statement to make things hopefully a bit clearer.